src/components/userGroups/UserGroup.scss

```diff
@@ -92,7 +92,9 @@
             fill: #999;
         }
     }
-
+    .info-Colour-bar-error-width{
+        width:307px;
+    }
```

Comment:
Is this just for width of error bar? If so then please override it in css file wherever it's used.